library/checkout/klarnacheckout.php

```diff
@@ -115,7 +115,7 @@ public function getShippingAddress()
     public function getArticles()
     {
         $products = $this->prepareProductArticles();
-        $wrappingVat = $this->buckarooConfigService->getConfigValue('klarna', 'wrapping_vat');
+        $wrappingVat = $this->buckarooConfigService->getConfigValue('klarnakp', 'wrapping_vat');
```

Comment:
do we need migrations for the setting as well?

Reply:
no we don`t need migration for this, it will get the id for klarnakp and check wrapping_vat with payment_id in configuration table

Reply:
No why?